pkg/kubeapiserver/authenticator/config.go

```diff
@@ -1,5 +1,6 @@
 /*
 Copyright 2014 The Kubernetes Authors.
+Copyright 2020 Authors of Arktos - file modified.
 
```

Comment:
Curious, why copyright statement is changed without any code change? Is it a make-up change for some miss is previous PRs?

Reply:
I probably have some change here and then removed. and the copyright sticks around.